README.md

```diff
@@ -369,6 +369,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Picasso](http://square.github.io/picasso/) - Image downloading and caching library for Android.
 * [Thumbnailator](https://github.com/coobird/thumbnailator) - Thumbnailator is a high-quality thumbnail generation library for Java.
 * [ZXing](https://github.com/zxing/zxing) - Multi-format 1D/2D barcode image processing library.
+* [TwelveMonkeys](https://github.com/haraldk/TwelveMonkeys) - An automatic plugin to the javax.imageio package to read and write to multiple image files that aren't supported by imageio, such as TIFF, PSD, SVG, ICNS and more.
```

Comment:
The description is grammatically incorrect and doesn't make any sense, just use the description in the project:

Collection of plugins which extend the number of image file formats supported using the `javax.imageio` package.